src/main.cpp

```diff
@@ -24,33 +27,49 @@ void loop() {
     connectMQTT(ESP_ID.c_str(), MQTT_USER,MQTT_PASSWORD);
   }
 
-
-
   delay(100);
 }
 
+/// Connect to wifi
+/// @ssid SSID of the wifi network you want to connect to.
+/// @wifiPassword Password of the wifi network.
 void connectWifi(char* ssid, char* wifiPassword)
 {
   WiFi.begin(ssid,wifiPassword);
 }
 
+/// Checks if the wifi is connected.
+/// returns: boolean that shows if the Wifi is connected.
 bool isWifiConnected()
```

Comment:
Deze functie is overbodig. Kan vervangen worden met een simpele Wifi.connected()

Reply:
Overigens, we hadden afgesproken variabelnamen camelcased maar begonnen met kleine letter dus wifi.. That said ik zou m liever wifiClient noemen gezien dat ook de class is die gebruikt wordt. Is net iets duidelijker.
pubSubClient is daarentegen wel goed.

Reply:
Door deze functie wordt de logica losgehaald van de library. Veel makkelijker om een mogelijke aanpassingen te maken en vind het zelf ook beter leesbaar en testbaar. De variable heet ook wifiClient.

Reply:
En dat zou een prima reden zijn... als er uberhaubt logica in deze functie zat. Maar nu is het puur een facade welke dezelfde functie op een andere plek aanroept. Dan kun je hem net zo goed gelijk aanroepen.